### ISSUE TITLE
Apollo ToDo

### ISSUE THREAD

**Issue by CobaltWolf** [CobaltWolf/Bluedog-Design-Bureau#328]
Cobalt's ToDo for Apollo
- [x] Fix engine Shroud texture
- [x] LEM Engine emissives
- [x] LES shroud
- [x] SLA normals
- [x] Resized SLA

**Comment by jsolson** [comment]
Lights are always on on block iii capsule.
B9 throws errors about alt textures missing.

**Comment by CobaltWolf** [comment]
There was something that I had to do that gave me reason to touch the SLA... but I can't remember what it was...

**Comment by jmrd98** [comment]
Was it the resize/rescale for 4.25 m base w/ 2.5 m top? feel free to shoot me if this has been done already.

**Comment by CobaltWolf** [comment]
err, yeah. I was saying, I had another reason to do something to it... But can't remember. But remembered that, I might as well do the rescale

**Comment by CobaltWolf** [comment]
OH! I remember. Wanted to fix the normals on the SLA so it actually looked smooth when it's closed.
